<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceConfig</class>
 <widget class="QDialog" name="FaceConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Faces</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Highlight Mode</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page_2"/>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>250</width>
               <height>184</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="3" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Curvature</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QDoubleSpinBox" name="sbCurvature">
                <property name="minimum">
                 <double>0.100000000000000</double>
                </property>
                <property name="maximum">
                 <double>0.500000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>0.200000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Decal Size</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Spacing</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="sbSpacing">
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>0.400000000000000</double>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="sbSpeed">
                <property name="maximum">
                 <double>5.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>0.200000000000000</double>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Movement Speed</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="sbDecalSize">
                <property name="minimum">
                 <double>0.100000000000000</double>
                </property>
                <property name="maximum">
                 <double>10.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>0.200000000000000</double>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Colors</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Curves</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FaceConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FaceConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbDecalSize</sender>
   <signal>valueChanged(double)</signal>
   <receiver>FaceConfig</receiver>
   <slot>OnDecalSizeChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbSpeed</sender>
   <signal>valueChanged(double)</signal>
   <receiver>FaceConfig</receiver>
   <slot>OnSpeedChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbSpacing</sender>
   <signal>valueChanged(double)</signal>
   <receiver>FaceConfig</receiver>
   <slot>OnSpacingChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbCurvature</sender>
   <signal>valueChanged(double)</signal>
   <receiver>FaceConfig</receiver>
   <slot>OnCurvatureChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnDecalSizeChanged(double)</slot>
  <slot>OnSpacingChanged(double)</slot>
  <slot>OnSpeedChanged(double)</slot>
  <slot>OnCurvatureChanged(double)</slot>
 </slots>
</ui>
